classes/PodsI18n.php

```diff
@@ -226,12 +226,12 @@ private static function default_strings() {
 	 */
 	public function get_current_language( $args = array() ) {
 
-		$args = wp_parse_args(
-			$args, array(
-				'refresh' => false,
-			)
+		$defaults = array(
```

Comment:
:no_entry_sign: Short array syntax must be used to define arrays

<sub>hash: 5659d632ad72cec2752380a50f6354aa</sub>